compiler/src/iree/compiler/Codegen/Common/TransformExtensions/CommonExtensions.cpp

```diff
@@ -132,31 +131,6 @@ struct FoldTensorEmptyExtract
   }
 };
 
-/// Trivial 1-1 pattern to retire once IREE adopts tensor.pack.
```

Comment:
@nicolasvasilache FYI that I'm going to drop most of usages about LinalgExt::PackOp/UnPackOp in this PR. The parent PR is still waiting for landing upstream patches. The ETA for landing this PR could be the end of next week.